parse_lib.py

```diff
@@ -106,6 +106,12 @@ def update_anchor_href(anchor):
         anchor['target'] = '_blank'
 
 
+def convert_to_img(html, equation):
```

Comment:
Sounds good!  I think your solution to the object-svg-detection problem is great!

At the risk of being pedantic, I want to emphasize again the point about abstractions because I am nervous, from the wording in your comment, that you may not fully understand what I mean.

You are right that "at the moment there's no application logic for deciphering between what is or isn't an SVG since there wasn't a need to make that distinction during this current version of the standard".  However, with regards to naming the lower-level function, this is irrelevant.  The lower-level `convert_svg_object_to_img` function is blissfully unaware of this problem, as should be the programmer reading the `convert_svg_object_to_img`.

You may have understood this already, but I wanted to be sure because I think it is very important.